### PR TITLE
Edwin

### PR DESCRIPTION
**Pull request by edgcastillo**

### CODE REVIEW

File: about_me_edwin.html [hunk 1]
```diff
@@ -0,0 +1 @@
+<h1>Hello!</h1>
```
Comment (nw0428) [inline about_me_edwin.html:1]: What is this file for?

**Comment by nw0428** [comment]
Its looking good but none of the pages link together and it looks like you might have at least one extra page.
You definitely need to have a path from the index page to your page(s).
Also, what does the email box do? Do I actually get registered for anything?

File: nmml_about_me/about_me.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+
```
Comment (nw0428) [inline nmml_about_me/about_me.html:3]: Is this your page? Where did it come from? If it is not your page/folder feel free to delete it and do a `git add -A` to have git record its deletion.

Comment (edgcastillo) [inline nmml_about_me/about_me.html:3]: I'm probably gonna change it today, had an idea and then change my mind.

Comment (nw0428) [inline nmml_about_me/about_me.html:3]: When you make a pull request it is to indicate that you are done.

I am going to go ahead and close this and you can reopen it when you think you are completely done.

Its looking nice btw =)

**Comment by nleoutsa** [comment]
Hi david,

The nmml_aboutme folder is mine. I pushed it before we met on Tuesday and
when people cloned the repo it was still there. Then you deleted it from
master and when people started doing pull requests, their branches still
had it... soooooo, fun topic on Wednesday, right?

-nick

**Comment by nw0428** [comment]
@nleoutsa Definitely! For now @Everyone if you did not write the code in that folder feel free (please) to delete it =)

**Pull request closed without merging**